tests/test_batched_covar2d.py

```diff
@@ -76,8 +77,15 @@ def test01(self):
         mean_bcov2d = self.bcov2d.get_mean()
         covar_bcov2d = self.bcov2d.get_covar(noise_var=noise_var)
 
-        self.assertTrue(np.allclose(mean_cov2d, mean_bcov2d))
-        self.assertTrue(self.blk_diag_allclose(covar_cov2d, covar_bcov2d))
+        self.assertTrue(np.allclose(
+            mean_cov2d,
+            mean_bcov2d,
+            atol=1e-5 if self.dtype == np.float32 else 1e-8))
```

Comment:
Eesh, now it's `1e-5`?

Reply:
Yes, I wasn't thrilled about this either, but that is what I found on the cross platform CI for our tests.